languageserver/server/server.go

```diff
@@ -492,7 +492,8 @@ func (s *Server) Hover(
 ) (*protocol.Hover, error) {
 
 	uri := params.TextDocument.URI
-	checker, ok := s.checkers[uri]
+	location := uriToLocation(uri)
+	checker, ok := s.checkers[location.ID()]
```

Comment:
These two lines seem to repeat in multiple places. Maybe could extract to a method?

Reply:
Good point, refactored in d6dd398169b87ad6909c8d673688ccdf86d10c53